View/AppShell.xaml.cs

```diff
@@ -1,9 +1,14 @@
-﻿namespace View;
+﻿using Model;
+namespace View;
+
 
 public partial class AppShell : Shell
 {
-    public AppShell()
+    public User User { get; private set; }
```

Comment:
`private set;` is niet nodig als je hem alleen in de constructor zet

Reply:
we laten hem lekker staan